react/src/components/ResourceAllocationFormItems.tsx

```diff
@@ -100,269 +89,51 @@ const ResourceAllocationFormItems: React.FC<
-        const configName =
-          {
-            'cuda.device': 'maxCUDADevicesPerContainer',
-            'cuda.shares': 'maxCUDASharesPerContainer',
-            'rocm.device': 'maxROCMDevicesPerContainer',
-            'tpu.device': 'maxTPUDevicesPerContainer',
-            'ipu.device': 'maxIPUDevicesPerContainer',
-            'atom.device': 'maxATOMDevicesPerContainer',
-            'warboy.device': 'maxWarboyDevicesPerContainer',
-          }[key] || 'cuda.device'; // FIXME: temporally `cuda.device` config, when undefined
-        result[key] = {
-          min:
-            parseInt(
-              _.filter(
-                currentImageAcceleratorLimits,
-                (supportedAcceleratorInfo) => {
-                  return supportedAcceleratorInfo?.key === key;
-                },
-              )[0]?.min as string,
-            ) || 0,
-          //
-          max: _.min([
-            baiClient._config[configName] || 8,
-            // scaling group all cpu (using + remaining), string type
-            !_.isEmpty(
-              // @ts-ignore
-              checkPresetInfo?.scaling_groups[currentResourceGroup]?.using?.[
-                key
-              ],
-            ) &&
-            !_.isEmpty(
-              // @ts-ignore
-              checkPresetInfo?.scaling_groups[currentResourceGroup]
-                ?.remaining?.[key],
-            )
-              ? _.toNumber(
-                  // @ts-ignore
-                  checkPresetInfo?.scaling_groups[currentResourceGroup]?.using[
-                    key
-                  ],
-                ) +
-                _.toNumber(
-                  // @ts-ignore
-                  checkPresetInfo?.scaling_groups[currentResourceGroup]
-                    ?.remaining[key],
-                )
-              : undefined,
-          ]),
-          remaining:
-            _.min([
-              // @ts-ignore
-              _.toNumber(checkPresetInfo?.keypair_remaining[key]),
-              // @ts-ignore
-              _.toNumber(checkPresetInfo?.group_remaining[key]),
-              // @ts-ignore
-              _.toNumber(checkPresetInfo?.scaling_group_remaining[key]),
-            ]) ?? Number.MAX_SAFE_INTEGER,
-        };
-        return result;
-      },
-      {} as {
-        [key: string]: {
-          min: number;
-          max: number;
-          remaining: number;
-        };
-      },
-    ),
 
-    session: {
-      min: 1,
-      max: _.min([
-        keypairResourcePolicy.max_concurrent_sessions,
-        3, //BackendAiResourceBroker.DEFAULT_CONCURRENT_SESSION_COUNT
-      ]),
-      remaining:
-        (keypairResourcePolicy.max_concurrent_sessions || 3) -
-        (keypair.concurrency_used || 0),
-    },
+  const sessionSliderLimitAndRemaining = {
+    min: 1,
+    max: _.min([
+      keypairResourcePolicy.max_concurrent_sessions,
+      3, //BackendAiResourceBroker.DEFAULT_CONCURRENT_SESSION_COUNT
+    ]) as number,
+    remaining:
```

Comment:
I think sessionSliderLimitAndRemaining also need to consider remaining resources as well as `derivedClusterSizeMaxLimit`, such as following CPU cores. Since max_concurrent_session goes more than max value, then remaining pin will go far away from slider.

Here's an example:
> - Current max_concurrent_sessions of user's resource policy is set to **30**.
> - User just created one session that allocates 1 CPU Core.

The containing values of sessionSliderLimitAndRemaining goes to,
```
{ 
   min: 1,
   max: 3, // since max_concurrent_sessions is 30, so max will be set to 3
   remaining: 29, // since max_concurrent_session exists, therefore 30 - 1(concurrency_used) goes to 29
}
```

### Screenshot(s)
<img width="2032" alt="Screenshot 2023-12-18 at 11 50 49 PM" src="https://github.com/lablup/backend.ai-webui/assets/46954439/200029e4-0e24-4cb3-bb71-d1764c88839c">
<img width="2032" alt="Screenshot 2023-12-18 at 11 50 53 PM" src="https://github.com/lablup/backend.ai-webui/assets/46954439/f5ed7bc7-ee9f-43b4-b44a-62fbc19928a7">